spec/json-spec.coffee

```diff
@@ -0,0 +1,13 @@
+describe "JSON grammar", ->
+  grammar = null
+
+  beforeEach ->
+    waitsForPromise ->
+      atom.packages.activatePackage("language-json")
+
+    runs ->
+      grammar = atom.grammars.grammarForScopeName("source.json")
+
+  it "parses the grammar", ->
```

Comment:
Since many (most?) other grammars use this exact wording, I'm gonna stick with the current naming unless there's a very compelling reason to change it :see_no_evil: